### ISSUE TITLE
basiliskii is on GitHub but using an unsafe source

### ISSUE THREAD

**Issue by waddlesplash** [haikuports/haikuports#1421]
```
Downloading: git+https://github.com/cebix/macemu.git#c2b519ee1ea848ac827e68cd1d536b3464b79ed1 ...
Error: Downloading from unsafe sources is disabled in haikuports.conf!
```

**Comment by Begasus** [comment]
This one can be closed I think :)

**Comment by korli** [comment]
Closed with 84257aab5e984428c113e921d623f9302915e17a
